internal/db/read_writer_test.go

```diff
@@ -2013,16 +2061,34 @@ func TestDb_LookupById(t *testing.T) {
 			name:       "simple-public-id",
 			underlying: db,
 			args: args{
-				resourceWithIder: user,
+				resource: user,
 			},
 			wantErr: false,
 			want:    user,
 		},
+		{
+			name:       "simple-non-public-non-private-id",
+			underlying: db,
+			args: args{
+				resource: accessory,
+			},
+			wantErr: false,
+			want:    accessory,
+		},
+		{
+			name:       "compond",
```

Comment:
Can we add a test to verify an error is returned when one or more values of a compound key are the zero value?

Reply:
ty!